crates/layout/src/layout_style.rs

```diff
@@ -66,12 +66,12 @@ impl Default for LayoutStyle {
             flex_grow: 0.0,
             flex_shrink: 0.0,
             flex_basis: Dimension::default(),
-            align_self: AlignSelf::default(),
-            align_content: AlignContent::default(),
-            align_items: AlignItems::default(),
-            flex_direction: FlexDirection::default(),
-            justify_content: JustifyContent::default(),
-            position_type: PositionType::default(),
+            align_self: AlignSelf::Auto,
+            align_content: AlignContent::Stretch,
+            align_items: AlignItems::Stretch,
+            flex_direction: FlexDirection::Row,
+            justify_content: JustifyContent::FlexStart,
+            position_type: PositionType::Relative,
         }
```

Comment:
So we're still using this? I thought this was going to be replaced with the layout proto.

Reply:
At least for this initial change, yes. The short version is that `LayoutStyle` is used in `DesignDoc`, so if we wanted to swap the existing `LayoutStyle` struct with the new, proto-generated `LayoutStyle` struct then we'd derive the Serde code for it (which is easy) and adjust code to deal with the differences. The amount of side work that was causing started to outweigh the benefits of replacing the existing struct. It was simpler to write conversion code.

We can revisit this decision soon. I'm finishing up the protobuf messages for everything `SerializedDesignDoc` uses and will start integrating them into the Rust code soon.

Reply:
I've not tried to measure performance of this change. If @iamralpht wants me to then I absolutely can, it'll just take more time. Do you have any benchmarking tests written for the rust code?

It's definitely less efficient to go serdeclass -> protoclass -> serialize... etc. I chose that for this change because this change is only a partial conversion of our structures that are serialized. I'll look back into whether we want to stay with wrapping the classes or just replace the Rust-native classes with the protobuf-generated classes when I finish the proto files for all of `serializedDesignDoc`.